tests/scripts/arm32_ci_test.sh

```diff
@@ -57,6 +60,12 @@ do
             exit_with_error "--buildConfig can be Debug, Checked or Release" true
         fi
         ;;
+    --ryujit)
+        if [[ "$__buildConfig" != "Release" ]]; then
+            exit_with_error "--ryujit is only avalable for Release by now" true
+        fi
+        COMPLUS_OPTIONS+=" COMPlus_AltJit=*"
```

Comment:
This should probably also set `COMPlus_AltJitNgen=*` for completeness.

Reply:
You are right. I forgot to add this because there is no readytorun test in current set.